src/EnergyPlus/OutputProcessor.cc

```diff
@@ -698,7 +698,7 @@ namespace OutputProcessor {
 		for ( Loop = MinIndx; Loop <= MaxIndx; ++Loop ) {
 			if ( ReqRepVars( Loop ).Key.empty() ) continue;
 			if ( ! SameString( ReqRepVars( Loop ).VarName, VariableName ) ) continue;
-			if ( ! DataOutputs::FindItemInVariableList( KeyedValue, VariableName ) ) continue;
+			if ( !RE2::FullMatch( KeyedValue, "(?i)" + ReqRepVars( Loop ).Key ) ) continue;
```

Comment:
I'll rely on @mbadams5 agreeing with this change [here](https://github.com/NREL/EnergyPlus/pull/6303#issuecomment-329193945)